server/resources/Category.py

```diff
@@ -1,74 +1,74 @@
-from flask import request
-from flask_restful import Resource
-from models import db, Category, CategorySchema
-from marshmallow import ValidationError
-
-categories_schema = CategorySchema(many=True)
-category_schema = CategorySchema()
-
-
-class CategoryResource(Resource):
-    def get(self):
-        categories = Category.query.all()
-        categories = categories_schema.dump(categories).data
-        return {'status': 'success', 'data': categories}, 200
-
-    def post(self):
-        json_data = request.get_json(force=True)
-        if not json_data:
-            return {'message': 'No input data provided'}, 400
-        # Validate and deserialize input
-        try:
-            data = category_schema.load(json_data)
-        except ValidationError as err:
-            return err.messages, 422
-
-        category = Category.query.filter_by(name=data['name']).first()
-
-        if category:
-            return {'message': 'Category already exists'}, 400
-        category = Category(
-            name=json_data['name']
-        )
-
-        db.session.add(category)
-        db.session.commit()
-
-        result = category_schema.dump(category).data
-        return {"status": 'success', 'data': result}, 201
-
-    def put(self):
-        json_data = request.get_json(force=True)
-        if not json_data:
-               return {'message': 'No input data provided'}, 400
-        try:
-            data = category_schema.load(json_data)
-        except ValidationError as err:
-            return err.messages, 422
-
-        category = Category.query.filter_by(id=data['id']).first()
-        if not category:
-            return {'message': 'Category does not exist'}, 400
-        category.name = data['name']
-        db.session.commit()
-
-        result = category_schema.dump(category).data
-
-        return {"status": 'success', 'data': result}, 204
-
-
-    def delete(self):
-        json_data = request.get_json(force=True)
-        if not json_data:
-            return {'message': 'No input data provided'}, 400
-        # Validate and deserialize input
-        try:
-            data = category_schema.load(json_data)
-        except ValidationError as err:
-            return err.messages, 422
-        category = Category.query.filter_by(id=data['id']).delete()
-        db.session.commit()
-
-        result = category_schema.dump(category).data
-
-        return {"status": 'success', 'data': result}, 204
+# from flask import request
```

Comment:
remove this commented code, if you need it again in the future, you can checkout this file from this specific comment